packages/zoltan2/example/block/kokkosBlock.cpp

```diff
@@ -75,14 +75,15 @@ int main(int narg, char *arg[]) {
   typedef Map_t::local_ordinal_type localId_t;
   typedef Map_t::global_ordinal_type globalId_t;
   typedef Tpetra::Details::DefaultTypes::scalar_type scalar_t;
+  typedef Tpetra::Map<>::node_type node_t;
 
   ///////////////////////////////////////////////////////////////////////
   // Generate some input data.
 
   int localCount = 40 * (rank + 1);
   int totalCount = 20 * nprocs * (nprocs + 1);
   int targetCount = totalCount / nprocs;
-  Kokkos::View<globalId_t*> globalIds("globalIds", localCount);
+  Kokkos::View<globalId_t*, Kokkos::HostSpace> globalIds("globalIds", localCount);
```

Comment:
The typical pattern for tests is to create device Views, but use `create_mirror_view` to get host mirrors, fill the host mirrors, then copy back to device.

Reply:
For MJ we decided to keep some things like IDs on HostSpace since we never use them in the kernels and this makes the current MPI communications more efficient. @kddevin I had converted this test to HostSpace for consistency with that change. This could probably use some further discussion when we go over these tests.